### ISSUE TITLE
Prison bus unable to remove nozzle

### ISSUE THREAD

**Issue by Moonhera** [echoroleplay/feedback#315]
**Description:** The prison bus is unable to have the nozzle removed when you try to refuel it. It is able to be fuelled but accessing the nozzle is not possible. When you pull away, it goes boom. Take a nap, and the nozzle is still unable to be removed. 

**Reproduction Steps:** Clearly describe the steps to reproduce the bug. 
1. Take prison bus to gas station
2. Put nozzle into the engine location
3. wait for it to be fuelled up
4. Go to remove the nozzle
5. Unable to remove

**Expected Behavior:** To be able to remove the nozzle

**Actual Behavior:** Unable to remove the nozzle at all. 

**Screenshot(s)/Video(s):**

**Comment by StuxxyOfficial** [comment]
I took a little look, seems like its just really particular on the targeting position to grab the nozzle, i assume when you had this issue, you couldn't see the prompt to remove it at all?

**Comment by Moonhera** [comment]
Not at all, other people struggled to get to it as well.

**Comment by StuxxyOfficial** [comment]
Yeah seems to be an issue, with just placing on that model, but if you aim at the door of the bus area or around the bus you should find the spot, though i will try find an alternative bone to whitelist the vehicle under so it will be a less problematic spot.
